.circleci/config.yml

```diff
@@ -394,7 +394,7 @@ jobs:
         default: ""
     machine:
       image: ubuntu-2204:current
-      resource_class: large
+      resource_class: xlarge
```

Comment:
This was taking 50m to build the CUDA wheel 🤯 .. hopefully the larger one will be much faster.